Extensions/Video/videotools.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * This is an example of some functions that can be used through events.
+ * They could live on any object but it's usual to store them in an object
+ * with the extension name in `gdjs.evtTools`.
+ *
+ * Functions are being passed the arguments that were declared in the extension.
+ *
+ * @memberof gdjs.evtTools
+ * @class video
+ * @static
+ * @private
+ */
+gdjs.evtTools.video = {};
```

Comment:
I think you can entirely remove this file, and reference to it in JsExtension.js :)